lib/Serverless.js

```diff
@@ -14,7 +14,7 @@ const Utils = require('./classes/Utils');
 const Service = require('./classes/Service');
 const Variables = require('./classes/Variables');
 const ConfigSchemaHandler = require('./classes/ConfigSchemaHandler');
-const ServerlessError = require('./classes/Error').ServerlessError;
+const ServerlessError = require('./serverless-error');
```

Comment:
I have a general question about naming patterns for utilities/modules like that. It seems like currently we mostly have camelCase where I've also noticed some utils with kebab-case. Which one is preferred?

Reply:
Very good question, I forgot to discuss that with you. Situation is as follows:

- `serverless` project since beginning practiced _camelCase_
- In Node.js community more common was to practise _kebab-case_. Afaik to avoid confusion between case-sensitive and case insensitive file systems, e.g.  now require'ing `serverless/lib/serverless.js` will work on case insensitive systems but will fail on case sensitive systems (with _kabab-case_ it's less likely someone will mix filename casing in require)
- Some other projects in this organization practice _kebab-case_
- As _kebab-case_ felt to me as more natural and safer choice, at some point I decided to shift `serverless` gradually towards _kabab-case_ (I didn't want to imply rename of all modules to avoid eventual breaking changes - but I think with new major we may simply rename all modules to it, to ensure same convention everywhere)

I don't have very strong opinion on that. What do you think?

Reply:
Thanks for detailed explanation, it makes sense to me :+1: